pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -108,6 +107,18 @@ public void testQueryOptions()
         h2Query, getH2Connection(), null, ImmutableMap.of("queryOptions", "useMultistageEngine=true"));
   }
 
+  @Test
+  public void testMultiValueColumnSelectionQuery()
+      throws Exception {
+    String pinotQuery =
+        "SELECT DivAirportIDs, DivAirports FROM mytable WHERE DATE_TIME_CONVERT(DaysSinceEpoch, '1:DAYS:EPOCH', "
+            + "'1:DAYS:SIMPLE_DATE_FORMAT:yyyy-MM-dd''T''HH:mm:ss.SSS''Z''', '1:DAYS') = '2014-09-05T00:00:00.000Z'";
+    String h2Query =
+        "SELECT DivAirportIDs__MV0, DivAirports__MV0 FROM mytable WHERE DaysSinceEpoch = 16318 LIMIT 10000";
+    ClusterIntegrationTestUtils.testQueryWithMatchingRowCount(pinotQuery, _brokerBaseApiUrl, getPinotConnection(),
+        h2Query, getH2Connection(), null, ImmutableMap.of("queryOptions", "useMultistageEngine=true"));
```

Comment:
i don't think we need integration test for this. we can simply add a test case in `pinot-query-runtime/src/test/resources/` 
but good to have this.

Reply:
i am not sure this counts are full support and will cause issue. for example if we do a group-by query. the behavior will be drastically different between
```
SELECT mvCol, count(*) FROM tbl GROUP BY mvCol  -- this will still behave as unnest
SELECT mvCol, count(*) FROM tblA JOIN tblB USING(key) GROUP BY mvCol  -- this will use the entire array as group
```
also the ordering of a group is not guaranteed --> maybe we are better off creating a ARRAY type in V1 that MV column can extend from.

Reply:
For multi stage, are we not keeping the implicit unnest behavior of MV columns when used in GROUP BY ?

Reply:
For simple group by, it's pushed down to v1 engine, so the behavior is same as v1. 

I think we need to support group by array in v1 then rewrite the query from v2.

Reply:
lets think about the behavior here.. lets keep implicit unnest on mv columns restricted to a single table and single level mv. with joins and windows lets make unnest explicit